src/components/Navbar.vue

```diff
@@ -8,49 +8,21 @@
             </button>
             <div class="collapse navbar-collapse" id="navbarNav">
                 <ul class="navbar-nav mr-auto">
-                    <li class="nav-item dropdown">
-                        <a class="nav-link dropdown-toggle" href="#" id="socialDropdown" role="button"
-                            data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                            Social
-                        </a>
-                        <div class="dropdown-menu light-navbar-dropdown" aria-labelledby="socialDropdown">
-                            <a class="dropdown-item" href="https://www.instagram.com/your_instagram"
-                                target="_blank">Instagram</a>
-                            <a class="dropdown-item" href="https://github.com/your_github" target="_blank">GitHub</a>
-                            <a class="dropdown-item" href="https://www.linkedin.com/in/your_linkedin"
-                                target="_blank">LinkedIn</a>
-                            <a class="dropdown-item" href="https://www.snapchat.com/add/your_snapchat"
-                                target="_blank">Snapchat</a>
-                            <div class="dropdown-divider"></div>
-                            <div class="dropdown-menu light-navbar-dropdown" aria-labelledby="moreSocialDropdown">
-                                <a class="dropdown-item" href="https://twitter.com/your_twitter" target="_blank">Twitter</a>
-                                <a class="dropdown-item" href="https://www.pinterest.com/your_pinterest"
-                                    target="_blank">Pinterest</a>
-                            </div>
-                        </div>
-                    </li>
                     <li class="nav-item dropdown">
                         <a class="nav-link dropdown-toggle" href="#" id="recipeDropdown" role="button"
                             data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                             Recipes
                         </a>
                         <div class="dropdown-menu light-navbar-dropdown" aria-labelledby="recipeDropdown">
-                            <a class="dropdown-item" href="#">Appetizers</a>
-                            <a class="dropdown-item" href="#">Main Courses</a>
-                            <a class="dropdown-item" href="#">Desserts</a>
-                            <a class="dropdown-item" href="#">Drinks</a>
-                            <a class="dropdown-item" href="#">More Recipes</a>
+                            <a class="dropdown-item"
+                                href="https://www.tasteofhome.com/collection/awesome-ideas-appetizer-recipes/">Appetizers</a>
```

Comment:
Kun je van alle nav items wellicht een array maken in een nl.json File bijv. met de i18n package.